code/modules/awaymissions/maploader/reader.dm

```diff
@@ -64,7 +64,7 @@ var/global/dmm_suite/preloader/_preloader = null
 		var/z_depth = length(zgrid)
 
 		//if exceeding the world max x or y, increase it
-		var/x_depth = length(copytext(zgrid,1,findtext(zgrid,"\n",2,0)))
+		var/x_depth = length(copytext(zgrid,1,findtext(zgrid,"\n",2,0)))/key_len
 		if(world.maxx<x_depth)

```

Comment:
_x_depth_ (the map width)  is used right below to calcule _y_depth_ (the map height).
So rather than dividing by _key_len_ here, either do it in the test (and the conditional assignement) or modify the _y_depth_ calculation.

Away missions have been stuck with small (read standard) maps for so long (no offense WJ, your maps are awesome anyway) this has been present since my rewriting of the maploader.
:+1: for fixing it.